memfault-ecs-cli/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir $PREFIX/bin
+cp $SRC_DIR/ecs-cli* $PREFIX/bin
+mv $PREFIX/bin/ecs-cli* $PREFIX/bin/ecs-cli
```

Comment:
this specific `mv` command doesn't look like it's doing anything?

Reply:
Agreed. That said, I stole this fully from https://github.com/memfault/conda-recipes/commit/21d361f6d29a4a78c709bc0e4c1d5e3379960e53

I'll try nixing it and seeing if it still builds alright

Reply:
Oh, now I know what it's doing. The actual file that gets plopped in there during `cp` is `ecs-cli.<platform>`. The `mv` renames it (and if you nix it the whole thing explodes on the chmod, obviously)

Lemme see if I can switch the name it gets initially, otherwise we'll keep it as is.

Reply:
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#filename did the trick. Removed the `mv` command